templates/default/components/Layout.jsx

```diff
@@ -29,14 +29,12 @@ function useTableOfContents(tableOfContents) {
     function onScroll() {
       const top = window.scrollY + 4.5;
       let current = headings[0].id;
-      // for (const heading of headings) {
       headings.map((heading) => {
         if (top >= heading.top) {
           current = heading.id;
         }
         return current;
       });
```

Comment:
Again, we don't need a `map` here as we only need to iterate over this array and not create a new one.